.github/workflows/ci.yml

```diff
@@ -1,3 +1,4 @@
+{% set default_python = '3.10' %}
```

Comment:
This shouldn't be making it to the rendered version should it? I suspect this is why the CI isn't running.

Reply:
it should not